frontend/src/services/authApi.ts

```diff
@@ -0,0 +1,29 @@
+import { createApi } from '@reduxjs/toolkit/query/react';
+import { TLoginDTO, TRegisterDTO } from '@/types/dtos';
+import axiosBaseQuery from './axiosBaseQuery';
+
+export const authApi = createApi({
+  reducerPath: 'authApi',
+  baseQuery: axiosBaseQuery({ baseUrl: 'http://10.0.2.2:5000' }),
```

Comment:
I can see this baseUrl string in multiple places in the code. Please move it to the file with constants.